### ISSUE TITLE
(web) service revision - add web service status (active, non-active)

### ISSUE THREAD

**Issue by lzehl** [openMetadataInitiative/openMINDS_core#509]
vocab (property name, value names) can be discussed and adjusted (names should be controlled terms?)

**Comment by lzehl** [comment]
@elenimath "operationStatus": linkedTypes for controlled vocab [active, non-active]?

@apdavison for the controlled vocab we probably have those already?

**Comment by apdavison** [comment]
We don't have terms for server status. The closest thing we have is [ActionStatusType](https://openminds-documentation.readthedocs.io/en/latest/instance_libraries/terminologies/actionStatusType.html), which is for processes that are expected to be relatively short-lived.

schema.org has [GameServerStatus](https://schema.org/GameServerStatus), which has terms suitable for digital services, I'm not sure they're suitable for human-provided services.

**Comment by lzehl** [comment]
@apdavison thanks for looking this up... Not sure if offline and online is what we want to state here? What status do we want to cover (dev/maintenance status OR integration/functionality)?

**Comment by apdavison** [comment]
I don't think we're interested in tracking the day-to-day status of services. I guess this is more about tracking whether the web service (version) is online and actively maintained, or if it is something that was online in the past but isn't any longer.

**Comment by lzehl** [comment]
What about the following?

"maintenanceState": linkedType MaintenanceState ["active" | "dormant" | others ?] 
"connectivityState": linkedType ConnectivityState ["online" | "offline" | others ?] 

I think these properties would cover different things which might be both useful?
For "active" vs. "dormant": What would be the definition/criteria?

**Comment by lzehl** [comment]
based on dev discussion:

"intendedOperationalStatus": ["available" | "unavailable" | "degraded"] 
terms provided as controlledTerms/OperationalStatus

**Comment by lzehl** [comment]
@apdavison would these also fit for software?

**Comment by apdavison** [comment]
> would these also fit for software?

I don't think so, it's too simple. For software, "actively developed" and "abandoned" are just two ends of a continuum; and this is just one axis for how usable software is.

**Comment by lzehl** [comment]
this will most likely go to the new deployment schema(s); should be decided with the overall service/software update.

**Comment by lzehl** [comment]
@apdavison @olinux @Raphael-Gazzotti I wonder if we can close this issue now? I think we will cover this in the deployment schema(s) now and I wonder if we can handle it there without a controlled terminology (through deployment start dates and end dates). Would you agree to close this issue?
